sdk/src/main/java/com/deploygate/sdk/DeployGate.java

```diff
@@ -145,6 +125,12 @@ public void run() {
             });
 
             mIsDeployGateAvailable = true;
+
+            mCustomLogInstructionSerializer.setDisabled(false);
```

Comment:
I've changed the timing to connect the remote service. It'd be great to do that right after getting a reply that looks like SYN.